lab-trent/lib/logfile.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports.readFiles = function(files, callback) {
```

Comment:
Single function module, so you don't need to give the function export a name; rather just assign directly to module.exports.